tools/sign.proj

```diff
@@ -0,0 +1,14 @@
+<Project DefaultTargets="Build">
```

Comment:
Can this also go under the signing directory?

Reply:
Yup.